packages/appstore/src/types/common.types.ts

```diff
@@ -185,4 +184,16 @@ export type TLinkedTo = {
     currency?: string;
 };
 
-export type TWalletAccount = NonNullable<ReturnType<typeof useWalletList>['data']>[number];
+export type TWalletAccount = {
```

Comment:
@farzin-deriv I've added this here because I'm using `TWalletAccount` type based on result `convertWallets` util function which receive result from `useWalletList` and add some properties for UI so like `icon`, `icon_type` and next. It's because when I added these properties inside of hook I needed to use util function from `@deriv/shared` which is not a dependency in `@deriv/hooks` so I needed to wait when `@deriv/utils` package will be released. Now I think I can add these properties inside of `useWalletList` hook

Reply:
@farzin-deriv anyway, in my opinion, it's better to have one common type `TWalletAccount` in package, it's more informative than `NonNullable<ReturnType<typeof useWalletList>['data']>[number]`. Let me know if you think differently